interfaces.go

```diff
@@ -201,6 +201,15 @@ type GasPricer interface {
 	SuggestGasPrice(ctx context.Context) (*big.Int, error)
 }
 
+// FeeHistory provides recent fee market data that consumers can use to determine
+// a reasonable maxPriorityFeePerGas value.
+type FeeHistory struct {
+	OldestBlock  *big.Int     `json:"oldestBlock"`             // block coresponding to first response value
+	Reward       [][]*big.Int `json:"reward,omitempty"`        // list every txs priority fee per block
+	BaseFee      []*big.Int   `json:"baseFeePerGas,omitempty"` // list of each block's base fee
+	GasUsedRatio []float64    `json:"gasUsedRatio"`            // ratio of gas used out of the total avaialble limit
+}
+
```

Comment:
We don't need the JSON field name declarations here because they will be handled by ethclient with its own internal type.

Reply:
removed in 3b5f23a43